tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/featured_case_study.html

```diff
@@ -0,0 +1,53 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% image featured_case_study_image fill-828x546 format-webp as featured_desktop_image %}
+{% image featured_case_study_image fill-1656x1092 format-webp as featured_desktop_image_retina %}
+{% image featured_case_study_image fill-860x646 format-webp as featured_mobile_image %}
+{% image featured_case_study_image fill-430x323 format-webp as featured_mobile_image_retina %}
+
+<div class="featured-case-study">
+    {# Image (with gradient if logo added) #}
+    <div class="featured-case-study__image-wrap{% if logo %} featured-case-study__image-wrap--with-logo{% endif %}">
+        <picture>
+            <source media="(max-width: 598px)" srcset="{{  featured_mobile_image.url }} 1x, {{ featured_mobile_image_retina.url }} 2x" />
+            <source media="(min-width: 599px)" srcset="{{ featured_desktop_image.url }} 1x, {{ featured_desktop_image_retina.url }} 2x" />
+            <img src="{{ featured_desktop_image.url }}" alt="" class="featured-case-study__image" />
+        </picture>
+
+        {# Optional logo #}
+        {% if featured_case_study_logo %}
+            {% image featured_case_study_logo fill-110x70 format-webp class="featured-case-study__company-logo" %}
```

Comment:
We need to discuss if we're going to add alt text for the logo images in the back-end - ideally we should I think. I missed it by mistake in the fieldspec.